### PR TITLE
Regression test for zeCommandListCreate()

### PR DESCRIPTION
**Pull request by glyons-intel**
Signed-off-by: Gail Lyons <gail.lyons@intel.com>

### CODE REVIEW

File: SYCL/Basic/regression/commandlist/gpu.cpp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// REQUIRES: L0_plugin
+// REQUIRES: gpu
+//
+// RUN: %clangxx -fsycl %S/Inputs/FindPrimesSYCL.cpp %S/Inputs/main.cpp -o %t.out -lpthread -lOpenCL
```
Comment (vladimirlaz) [inline SYCL/Basic/regression/commandlist/gpu.cpp]: I am not sure that *-lOpenCL* is needed

File: SYCL/Basic/regression/commandlist/gpu.cpp [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// REQUIRES: L0_plugin
```
Comment (vladimirlaz) [inline SYCL/Basic/regression/commandlist/gpu.cpp]: there test will not work as we do not have L0_plagin feature defined in lit.cfg.py

File: SYCL/Basic/regression/commandlist/gpu.cpp (hunk 1 above)
Comment (glyons-intel) [reply SYCL/Basic/regression/commandlist/gpu.cpp]: good catch.  i removed it.

File: SYCL/Basic/regression/commandlist/gpu.cpp (hunk 2 above)
Comment (glyons-intel) [reply SYCL/Basic/regression/commandlist/gpu.cpp]: cruft.  sorry about that.  i removed it.

**Review by vladimirlaz — APPROVED** [review]

**Review by smaslov-intel — APPROVED** [review]

**Pull request merged**